bundles/org.openhab.binding.modbus.lambda/src/main/java/org/openhab/binding/modbus/lambda/internal/LambdaBindingConstants.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.lambda.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.modbus.ModbusBindingConstants;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link LambdaBindingConstants} class defines common
+ * constants, which are used across the whole binding.
+ *
+ * @author Paul Frank - Initial contribution
+ * @author Christian Koch - modified for lambda heat pump based on stiebeleltron binding for modbus
+ */
+@NonNullByDefault
+public class LambdaBindingConstants {
+
+    private static final String BINDING_ID = ModbusBindingConstants.BINDING_ID;
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_LAMBDAHP = new ThingTypeUID(BINDING_ID, "lambda-heatpump");
+
+    // Channel group ids
+    public static final String GROUP_GENERAL_AMBIENT = "generalAmbient";
+    public static final String GROUP_GENERAL_EMANAGER = "generalEManager";
+    public static final String GROUP_HEATPUMP1 = "Heatpump1";
+    public static final String GROUP_HEATPUMP1SET = "Heatpump1Set";
+    public static final String GROUP_BOILER1 = "Boiler1";
+    public static final String GROUP_BOILER1MT = "Boiler1Mt";
+    public static final String GROUP_BUFFER1 = "Buffer1";
+    public static final String GROUP_BUFFER1MT = "Buffer1Mt";
+    public static final String GROUP_HEATINGCIRCUIT1 = "HeatingCircuit1";
+    public static final String GROUP_HEATINGCIRCUIT1SETTING = "HeatingCircuit1Setting";
```

Comment:
Please adjust to the naming convention `lower-case-hypen`

For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention